users/user.rb

```diff
@@ -2,24 +2,26 @@
 class User
     # include page_collection do i need this? require/include..
     # i think i dont but lets see
+    attr_accessor :current_page
```

Comment:
Should we have it `attr_reader`? Doesn't seem like anything outside the class should be writing pages at this time?

Reply:
100% right, will fix that
That's what I meant to write

Reply:
Addressed in next pull